src/features/usersSlice.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { getUsers } from '../api/users';
+// eslint-disable-next-line import/no-cycle
+import { RootState } from '../app/store';
+import { User } from '../types/User';
```

Comment:
try to fix that error instead of disable it

Reply:
i fixed adding comments with spaces? its cuz demo didn`t update